### ISSUE TITLE
Auto Upload going haywire

### ISSUE THREAD

**Issue by diemelone** [nextcloud/android#8110]
I have installed the nextcloud app on my smartphone and was curious so I tried out the features. I was previously using owncloud and Auto Upload never seemed to work. I have activated the Auto Upload three days ago and deactivated it almost immediately (clicking the cloud icon in the folder selection).
That seemed to work. Today I used the app again and it started uploading every picture from that on folder which was still deactivated, no idea why...
Please make a checkbox before selecting the folders as in the manual or in the current version of owncloud
[https://docs.nextcloud.com/android/android_app.html](https://docs.nextcloud.com/android/android_app.html)

### Steps to reproduce
1. Doing nothing
2. 
3. 

### Expected behaviour
- no Auto Upload

### Actual behaviour
- every picture is uploaded

### Can you reproduce this problem on https://try.nextcloud.com?
- didn't try for privacy reasons

### Environment data
Android version: 10

Device model: Samsung M31

Stock or customized system: stock

Nextcloud app version: 3.14.1

Nextcloud server version: 20.0.5

Reverse proxy: ?

#### Web server error log
```
? is this necessary and if so, which parts should be deleted?
```

#### Nextcloud log (data/nextcloud.log)
```
? is this necessary and if so, which parts should be deleted?
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Is the cloud symbol on *all* auto upload folder disabled? 
We never had such a situation that pictures are uploaded without explicitly enabling it…

**Comment by diemelone** [comment]
of course, my quick solution was reinstalling the nc app and not opening the Auto Upload settings page at all

**Comment by tobiasKaminsky** [comment]
Strange, but glad to hear that it works now as expected after re-installation.

**Comment by diemelone** [comment]
Closed, is there a fix yet?
The Problem is easily reproducible, at least for me. I just added a folder to sync and deactivated syncing 10 mins later. The whole folder tried to sync as before (although syncing was disabled). 
That just proofs that there is an issue...
Please just add the checkbox or similar again... you had a working solution.

**Comment by tobiasKaminsky** [comment]
Sorry, I misunderstood your previous comment.

**Comment by jeffWelling** [comment]
Random thought from left field.. 
If you tell it to sync a folder, and then deactivate the sync, you see it still syncing right?
What if you then reboot your phone? When it starts back up again, does the sync process resume?
